ArkEcosystem/laravel-ui
# fix: normalize external link component

## Summary

https://app.clickup.com/t/u3rqze


Dependants
1. https://github.com/ArkEcosystem/marketsquare.io/pull/2207
2. https://github.com/ArkEcosystem/ark.io/pull/978

Fixes some differences between external links.

**Notes:**
1. Part of the problem is that the markdown parser removes the `span` inside the external links leaving an extra blank space so I am whitelisting that tag on the markdown parser:

![Screen Shot 2021-08-12 at 13 08 31](https://user-images.githubusercontent.com/17262776/129247043-a29c5307-0cec-4ca0-ab07-020de6de27b6.png)

2. Fixed a regression with long links not breaking again (fixed on the screenshot)
![image](https://user-images.githubusercontent.com/17262776/129247279-63a31143-798c-415d-af28-5febf5a2d52e.png)

3. Removes some attributes that don't make sense like the `inline` (all links are inline) and the `allowWrap` and checked all the projects that are affected (only ark.io and msq were affected) 

## To test

Merge this on the msq PR and on the ark.io PR, recompile assets and compare the links on articles, user profile, and basically, everything that is dynamic and the links added manually like the link on the footer on ark.io or the social network links on the projects.

Ensure to review every possible link you can find (search for `<x-ark-external-link` references)
 

## Checklist
